docs/source/share_kedro_viz.md

```diff
@@ -23,11 +23,12 @@ When prompted for a project name, you can enter anything, but we will assume `Sp
 
 When your project is ready, navigate to the root directory of the project.
 
-## Update and install the dependencies
+## Publish and share Kedro-Viz automatically
 
-Kedro-Viz requires specific minimum versions of `fsspec[s3]`, and `kedro` to publish your project.
+You can automate the process of publishing and sharing your Kedro-Viz. This section describes the steps for AWS, which is the only cloud provider supported for automation at present. Integration with other cloud providers, namely Azure and GCP, will be added soon. In the absence of automated publish and share for other platforms, there is a [manual, platform-agnostic publish and share process](#platform-agnostic-sharing-with-kedro-viz) described below. You can use the manual process for sharing on static website hosts like GitHub pages, and cloud providers like Azure and GCP.
```

Comment:
This paragraph is a bit dense. Maybe it could be said "there are two ways to publish and share your Kedro-Viz: 1. the automatic way (only AWS is supported at present) and 2. the manual way (you can use any static hosting or object storage of your choosing)"

Reply:
Also: "share your Kedro-Viz" should be replaced by "share your Kedro-Viz project", right? I recall there was a discussion around terminology a while ago cc @stichbury

Reply:
I think it's a valid comment but equally is fine as it is (partly because I wrote it) so happy for Rashida to merge as is or make some further changes, depends on urgency to get this release out.